src/FileConventions/Library.fs

```diff
@@ -238,7 +249,13 @@ let private WrapParagraph (text: string) (maxCharsPerLine: int) : string =
 
 let WrapText (text: string) (maxCharsPerLine: int) : string =
     let wrappedParagraphs =
-        text.Split $"{Environment.NewLine}{Environment.NewLine}"
+        text.Split(
+            [|
+                $"{UnixEol}{UnixEol}"
+                $"{Environment.NewLine}{Environment.NewLine}"
```

Comment:
why using UnixEol and Environment.NewLine instead of UnixEol and WindowsEol?

Reply:
I made it more consistant

Reply:
@aarani now that we are making changes to this, let's also make sure it works in macOS, and to do that I'd prefer if we adopt a common function instead of violating DRY, let's do something similar to this: https://github.com/nblockchain/conventions/commit/2ccc084d8187913decfac4b7f893010612b5bfce